Enterprise-CMCS/cmcs-eregulations
# EREGCSC-2756 - Update swagger

Resolves # EREGCSC-2756

**Description-**
Update swagger so that it does not return an error anymore. The issue was that ResourceGroupSerializer needed to be updated to serialize the entire resource not just using the primary keys. 

**This pull request changes...**

- expected change 1

**Steps to manually verify this change...**

1. steps to view and verify change

visit : [swagger](https://fdc0l1ak6c.execute-api.us-east-1.amazonaws.com/dev1407/api/swagger)
notice error is no longer there. 
2. on your local go to this pull request branch.
3. on your local, run django app `make local`
4. run `run make local.spectacular`
5. check to see that there are no errors.
